pytest/tests/mocknet/mirror.py

```diff
@@ -548,10 +593,20 @@ def run_remote_cmd(args, traffic_generator, nodes):
                 f'cannot give --chain-id --start-height or --unique-id along with --local-test'
             )
         traffic_generator, nodes = local_test_node.get_nodes()
+        node_config.configure_nodes(nodes + [traffic_generator],
+                                    node_config.TEST_CONFIG)
     else:
         if args.chain_id is None or args.start_height is None or args.unique_id is None:
             sys.exit(
                 f'must give all of --chain-id --start-height and --unique-id')
         traffic_generator, nodes = remote_node.get_nodes(
             args.chain_id, args.start_height, args.unique_id)
-    args.func(args, traffic_generator, nodes)
+        node_config.configure_nodes(nodes + [traffic_generator],
```

Comment:
If you chose to move the node identification to an attribute then maybe it is worth merging traffic node with the other nodes and add another config for it.